doc/Media2.xml

```diff
@@ -2167,6 +2167,244 @@
         </variablelist>
       </section>
     </section>
+	<section xml:id="section_k4m_bcd_yyz">
+      <title>Audio Clip Management</title>
+      <section>
+        <title>Audio Clip Configuration</title>
+        <para>This section is about uploading one or more pre-recorded audio clips to the device. The uploaded audio clips can be played by the user on an as-needed basis, on a configured schedule, or when the associated event is triggered. The audio clip configuration includes the following parameters:</para>
+        <itemizedlist>
+          <listitem>
+            <para>Enabled [xs:boolean] – Enables/disables the audio clip.</para>
+          </listitem>
+          <listitem>
+            <para>Name [xs:string] – Name of the audio clip.</para>
+          </listitem>          
+		  <listitem>
+            <para>AudioOutputToken [tt:ReferenceToken] optional, unbounded – Optional token list of physical audio outputs. If no AudioOutputToken is provided, the device should play the audio clip through all physical audio outputs.</para>
+          </listitem>
+          <listitem>
+            <para>Type [xs:string] - Format of the audio clip. Supported audio Clip formats are WAV, MP3, OGG-Opus, OGG-Vorbis, OGG-Flac, OGG-Speex.</para>
+          </listitem>          
+          <listitem>
+            <para>RepeatCycles [xs:int] - The number of times the audio clip can be repeated when it is played. A value of 0 means no repeat (i.e., the audio clip plays once and does not repeat). A value of 1 means the audio clip plays once and repeats once after the first play. A value of -1 means the audio clip repeats infinitely after the first play until it is stopped by the user.</para>
+          </listitem>
+		  <listitem>
+            <para>RepeatInterval [xs:int] optional - Optional time interval, in seconds, between audio clips when the audio clip is repeated.</para>
+          </listitem>
+		  <listitem>
+            <para>AudioOutputLevel [xs:int] optional - Optional level to control the volume of the audio clip when it is played.</para>
```

Comment:
If this value corresponds to a range defined in OutputLevelRange of AudioOutputConfigurationOptions?
If yes:
1) Maybe it is better to use the same field name as we have in DeviceIO service (OutputLevel, but not AudioOutputLevel)?
2) Can device reject request in case AudioOutputLevel is out of OutputLevelRange (I didn't see such error in Add/Set clip commands) or it shall silently adjust it?

Reply:
1. OutputLevel parameter listing in AudioOutputConfigurationOptions is reasonable since the context of the API already related to AudioOutput. AudioOutputLevel  parameter is defined in Audio Clip Configuration, just mentioning OutputLevel here does not give full context as AudioOuput.
2. Both Add/Set Audio clip have the fault code,
env:Sender - ter:InvalidArgVal - ter:InvalidConfig
The configuration parameters are not possible to set.
3. If the multiple audio output tokens given in Add/Set Audio clip, the client wants to play the audio clip in multiple physical audio outputs of the device.